_data/people.yml

```diff
@@ -220,7 +220,21 @@ freiburg:
         location_link: "https://goo.gl/maps/zEL3g3KVE8Q2"
         gitter: pavanvidem
         orcid: 0000-0002-5192-126X
-      
+
+    sanjaysrikakulam:
+        name: Sanjay Kumar Srikakulam
+        title: M.Sc. Bioinformatics, Researcher, DevOps
+        email: srikakus@informatik.uni-freiburg.de
+        location: "Build.: 079, Room: -1005"
+        location_link: "https://goo.gl/maps/zEL3g3KVE8Q2"
+        gitter: sanjaysrikakulam
+        matrix: "@sanjaysrikakulam:matrix.org"
+        twitter: SanjaySrikakul
+        linkedin: bjorn-grüning-2b84a895
```

Comment:
That you probably want to change.

Reply:
Oops! Sorry, fixed it with a new commit.